README.md

```diff
@@ -1,61 +1,57 @@
 # SyncPromise
 
-A fast, small, _safe_ promise implementation with synchronous promise resolution
-and an API which resembles ECMAScript promises.
+A fast, small, _safe_ promise implementation with synchronous promise
+resolution and an API which resembles `ECMAScript` promises.
```

Comment:
I've seen it be used this way, but sure, I can remove, as yes, it is technically not code.